docs/getting-started/configuration/composition.yaml

```diff
@@ -150,8 +195,12 @@ spec:
                 fmt: "%s-postgresql"
         - fromFieldPath: "spec.parameters.storageGB"
           toFieldPath: "spec.forProvider.allocatedStorage"
+        - fromFieldPath: "spec.parameters.passwordSecretName"
```

Comment:
Name and namespace should be patched in:

```yaml
       - fromFieldPath: "spec.parameters.passwordSecretRef.name"
          toFieldPath: "spec.forProvider.passwordSecretRef.name"
        - fromFieldPath: "spec.parameters.passwordSecretRef.namespace"
          toFieldPath: "spec.forProvider.passwordSecretRef.namespace"
```

Reply:
@stevendborrelli thanks a lot for the catch and apologies, I did a password ref extension last minute. 

https://github.com/upbound/universal-crossplane/pull/279/commits/59b34b1591fe25788e43cd411da2213b65ef2e7d should fix it